x-pack/plugins/index_management/public/application/sections/home/index_list/details_page/details_page_mappings.tsx

```diff
@@ -0,0 +1,167 @@
+import {
+  EuiButton,
+  EuiCodeBlock,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiLink,
+  EuiPageTemplate,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+import { css } from '@emotion/react';
+import { RouteComponentProps } from 'react-router-dom';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { SectionLoading } from '@kbn/es-ui-shared-plugin/public';
+import { useLoadIndexMappings, documentationService } from '../../../../services';
+
+export const DetailsPageMappings: FunctionComponent<RouteComponentProps<{ indexName: string }>> = ({
+  match: {
+    params: { indexName },
+  },
+}) => {
+  const { isLoading, data, error, resendRequest } = useLoadIndexMappings(indexName);
+
+  if (isLoading) {
+    return (
+      <SectionLoading>
+        <FormattedMessage
+          id="xpack.idxMgmt.indexDetails.mappings.loadingDescription"
+          defaultMessage="Loading index mappings…"
+        />
+      </SectionLoading>
+    );
+  }
+  if (error) {
+    return (
+      <EuiPageTemplate.EmptyPrompt
+        data-test-subj="indexDetailsMappingsError"
+        color="danger"
+        iconType="warning"
+        title={
+          <h2>
+            <FormattedMessage
+              id="xpack.idxMgmt.indexDetails.mappings.errorTitle"
+              defaultMessage="Unable to load index mappings"
+            />
+          </h2>
+        }
+        body={
+          <>
+            <EuiText color="subdued">
+              <FormattedMessage
+                id="xpack.idxMgmt.indexDetails.mappings.errorDescription"
+                defaultMessage="There was an error loading mappings for index {indexName}: {error}"
+                values={{
+                  indexName,
+                  error: error.error,
+                }}
+              />
+            </EuiText>
+            <EuiSpacer />
+            <EuiButton
+              iconSide="right"
+              onClick={resendRequest}
+              iconType="refresh"
+              color="danger"
+              data-test-subj="indexDetailsMappingsReloadButton"
+            >
+              <FormattedMessage
+                id="xpack.idxMgmt.indexDetails.mappings.reloadButtonLabel"
+                defaultMessage="Reload"
+              />
+            </EuiButton>
+          </>
+        }
+      />
+    );
+  }
+
+  return (
+    // using "rowReverse" to keep docs links on the top of the mappings code block on smaller screen
+    <EuiFlexGroup
+      wrap
+      direction="rowReverse"
+      css={css`
+        height: 100%;
+      `}
+    >
+      <EuiFlexItem
+        grow={1}
+        css={css`
+          min-width: 400px;
+        `}
+      >
+        <EuiPanel grow={false} paddingSize="l">
+          <EuiFlexGroup alignItems="center" gutterSize="s">
+            <EuiFlexItem grow={false}>
+              <EuiIcon type="iInCircle" />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiText>
+                <b>
```

Comment:
I used `EuiTitle` for this in my [stats PR](https://github.com/elastic/kibana/pull/165027/files#diff-7248628d8a72e038281b71d658bc05509b240f100040cb418aa2a4576701b71fR117). Do you think that's better for a11y purposes? 🤔

Reply:
Good idea, thanks! I'll use the same